### ISSUE TITLE
Add ability to download NetCDF data as CSV

### ISSUE THREAD

**Issue by HansVraga** [USGS-WiM/FEV#185]
Once https://github.com/USGS-WiM/STNServices/issues/2 is complete, add option for user to make use of this endpoint to data downloads.

**Comment by HansVraga** [comment]
Switched to deferred because prerequisite service work is deffered at this time
